code/modules/surgery/organs/eyes.dm

```diff
@@ -405,3 +409,23 @@
 	name = "apid eyes"
 	desc = "Designed for navigating dark hives, these eyes have improvement to low light vision."
 	see_in_dark = 8
+
+/obj/item/organ/eyes/psyphoza
+	name = "psyphoza eyes"
+	desc = "Conduits for psychic energy, hardly even eyes."
+	icon_state = "psyphoza_eyeballs"
+	actions_types = list(/datum/action/item_action/organ_action/psychic_highlight)
+	see_in_dark = 8
+	lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE
+	blind_type = /atom/movable/screen/fullscreen/blind/psychic
+	sight_flags = SEE_MOBS | SEE_OBJS | SEE_TURFS
+	can_see = FALSE
+
+/obj/item/organ/eyes/psyphoza/Insert(mob/living/carbon/M, special, drop_if_replaced, initialising)
+	. = ..()
+	M.become_blind("uncurable", /atom/movable/screen/fullscreen/blind/psychic, FALSE)
+	M.remove_client_colour(/datum/client_colour/monochrome/blind)
```

Comment:
This shouldn't be added in the first place, client color is finnicky you don't wanna mess with it like this